packages/angular_devkit/architect/src/architect.ts

```diff
@@ -254,14 +259,18 @@ export class Architect {
     return new builderClass(context);
   }
 
+  getSchema(schemaPath: Path): Observable<JsonObject> {
+    return this._host.read(schemaPath).pipe(
+      concatMap((buffer) => of(JSON.parse(virtualFs.fileBufferToString(buffer)))),
```

Comment:
`map`?

Reply:
Is there a benefit in using `map` over `concatMap` here?

Reply:
Remove the `of()`?